lessons/Data-Science/histograms/langs/en-us/index.adoc

```diff
@@ -244,7 +244,7 @@ Before we shift our focus to your chosen datasets, let’s quickly review what w
 @A{Histograms display a single column of quantitative data.}
 
 @Q{How is a histogram different from a bar chart?}
-@A{Because a bar chart displays categorical data, we can rearrange the bars in any order we wish. Because the quantitative data of a histogram must follow a natural order, bars cannot be rearranged.}
+@A{Because a bar chart visualizations categorical data, we can rearrange the bars in any order we wish. Because the quantitative data of a histogram must follow a natural order, bars cannot be rearranged.}
```

Comment:
```suggestion
@A{Because a bar chart displays categorical data, we can rearrange the bars in any order we wish. Because the quantitative data of a histogram must follow a natural order, bars cannot be rearranged.}
```